packages/@vue/cli-plugin-e2e-cypress/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "@vue/cli-shared-utils": "^4.2.2",
-    "cypress": "^3.8.3",
+    "cypress": "^4.0.0",
```

Comment:
Should be 4.0.1

Reply:
we released a quick release patch, will update. 👍